locales/de/pdk.po

```diff
@@ -1,604 +0,0 @@
-# German translations for puppet development kit package.
```

Comment:
I would be interested how the non-up-to-date de.po file causes issues with threading, especially outside the `de` locale.

`rake gettext:update_pot` and `rake gettext:po[de]` worked fine until now to update the po file as needed, and to the right thing for gettext. I'm sure transifex can one-time import the existing translations once we get around to integrating.

> Another potential solution would be for you to keep the German file locally and not checked in to the repo.

I'm not even using the thing, what value would it have on my local machine?